src/ServiceControlInstaller.Engine/Interfaces.cs

```diff
@@ -73,7 +73,16 @@ public interface IDatabaseMaintenanceSupport : IVersionInfo
         int? DatabaseMaintenancePort { get; }
     }
 
-    public interface IServiceControlAuditInstance : IServiceInstance, IServiceControlPaths, IHttpInstance, IInstallable, IDatabaseMaintenanceSupport, ITransportConfig
+    public interface IServiceControlBaseInstance : IHttpInstance, 
```

Comment:
Looks like this interface is nowhere really needed but here. So this is really just convenience to make sure we don't have to declare all those types?

Reply:
The interface is needed for removing a bunch of validation logic. Currently, it is duplicated for Monitoring/SC/SCA because of the lack of common interface